### PR TITLE
Small corrections and example for auto-encryption

### PR DESCRIPTION
**Pull request by eco-minio**
## Checklist:

- [ ] My code follows the code style of this project.
- [ ] My change requires a change to the documentation.
- [X] I have updated the documentation accordingly.
- [ ] I have added unit tests to cover my changes.
- [ ] I have added/updated functional tests in [mint](https://github.com/minio/mint). (If yes, add `mint` PR # here: )
- [ ] All new and existing tests passed.

### CODE REVIEW

File: docs/kms/README.md [hunk 1]
```diff
@@ -1,28 +1,29 @@
...
    treated as compromised.
 
 **Important:**  
-If multiple minio server are configured as [gateways](https://github.com/minio/minio/blob/master/docs/gateway/README.md)
-pointing to the *same* backend - for example the same NAS storage - than the KMS configuration **must** be equal for
-all gateways. Otherwise one gateway may not be able to decrypt objects created by another gateway. It's the operators 
-responsibility to ensure that.
+If multiple Minio servers are configured as [gateways](https://github.com/minio/minio/blob/master/docs/gateway/README.md)
+pointing to the *same* backend - for example the same NAS storage - then the KMS configuration **must** be the same for
+all gateways. Otherwise one gateway may not be able to decrypt objects created by another gateway. It is the operators
```
Comment (kannappanr) [inline docs/kms/README.md]: Should it be `operator's`

File: docs/kms/README.md [hunk 2]
```diff
@@ -43,7 +44,8 @@ Minio requires the following Vault setup:
 
 **2.1.1 Start Vault server in dev mode**
 
-In dev mode, Vault server runs in-memory and starts unsealed. Note that running Vault in dev mode is insecure and any data stored in the Vault is lost upon restart.
+In dev mode, Vault server runs in memory and starts unsealed. Note that running Vault in dev mode is insecure and any data stored in the Vault is lost upon restart.
```
Comment (poornas) [inline docs/kms/README.md]: in-memory is better, it is an actual term

**Review by poornas — CHANGES_REQUESTED** [review]

**Review by deekoder — DISMISSED** [review]

**Review by kannappanr — APPROVED** [review]
LGTM

**Review by poornas — APPROVED** [review]
LGTM

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Comment by minio-ops** [comment]
## Mint Automation
| Test | Result |
|:--- |:--- |
| mint-tls.sh | :heavy_check_mark: |
| mint-compression-xl.sh | :heavy_check_mark: |
| mint-xl.sh | :heavy_check_mark: |
| mint-large-bucket.sh | :heavy_check_mark: |
| mint-compression-fs.sh | :heavy_check_mark: |
| mint-worm.sh | :heavy_check_mark: |
| mint-fs.sh | :heavy_check_mark: |
| mint-dist-xl.sh | :heavy_check_mark: |

**Pull request merged**